File: app/src/main/java/com/jawnnypoo/openmeh/util/ColorUtil.java
package com.jawnnypoo.openmeh.util;

import android.content.res.ColorStateList;

/**
 * Color util!
 */
public class ColorUtil {

    public static ColorStateList createColorStateList(int color, int pressed) {
        return new ColorStateList(new int[][]{
                new int[]{android.R.attr.state_pressed},
                new int[]{}
        }, new int[]{
                pressed,
                color
        });
    }
}

File: app/src/main/java/com/jawnnypoo/openmeh/util/ColorUtil.kt
package com.jawnnypoo.openmeh.util

import android.content.res.ColorStateList

/**
 * Color util!
 */
object ColorUtil {

    fun createColorStateList(color: Int, pressed: Int): ColorStateList {
        return ColorStateList(arrayOf(intArrayOf(android.R.attr.state_pressed), intArrayOf()), intArrayOf(pressed, color))
    }
}
